streaming/java/BUILD.bazel

```diff
@@ -213,6 +213,8 @@ genrule(
         for f in $(locations //streaming/java:all_java_proto); do
             unzip -q -o "$$f" -x META-INF/MANIFEST.MF -d "$$WORK_DIR/streaming/java/streaming-runtime/src/main/java"
         done
+        # remove third party protobuf
+        rm -rf $$WORK_DIR/streaming/java/streaming-runtime/src/main/java/com/
```

Comment:
add protobuf to path?

Reply:
If we add `com` pakcage code to our repo, this will delete those code and ius hard to figure out

Reply:
changed to `com/google/protobuf/`